### ISSUE TITLE
Django 1.9 causes "Apps aren't loaded yet" error

### ISSUE THREAD

**Issue by fsto** [makingspace/transmissions#8]
Steps to Reproduce:
1. `pip install django==1.9`
2. `pip install git+git://github.com/makingspace/transmissions.git@aa4927df903bac80b496dbc487c3550f7d84becf`
3. `django-admin startproject main`
4. Add `transmissions` to your `installed_apps`
5. `python main/manage.py runserver`

Actual Results:

```
Traceback (most recent call last):
  File "main/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/moje/dev/test/env/lib/python2.7/site-packages/django/core/management/__init__.py", line 350, in execute_from_command_line
    utility.execute()
  File "/Users/moje/dev/test/env/lib/python2.7/site-packages/django/core/management/__init__.py", line 342, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/moje/dev/test/env/lib/python2.7/site-packages/django/core/management/__init__.py", line 176, in fetch_command
    commands = get_commands()
  File "/Users/moje/dev/test/env/lib/python2.7/site-packages/django/utils/lru_cache.py", line 100, in wrapper
    result = user_function(*args, **kwds)
  File "/Users/moje/dev/test/env/lib/python2.7/site-packages/django/core/management/__init__.py", line 71, in get_commands
    for app_config in reversed(list(apps.get_app_configs())):
  File "/Users/moje/dev/test/env/lib/python2.7/site-packages/django/apps/registry.py", line 137, in get_app_configs
    self.check_apps_ready()
  File "/Users/moje/dev/test/env/lib/python2.7/site-packages/django/apps/registry.py", line 124, in check_apps_ready
    raise AppRegistryNotReady("Apps aren't loaded yet.")
django.core.exceptions.AppRegistryNotReady: Apps aren't loaded yet.
```

Expected Results:

```
python manage.py runserver
Performing system checks...

System check identified no issues (0 silenced).

You have unapplied migrations; your app may not work properly until they are applied.
Run 'python manage.py migrate' to apply them.

August 25, 2016 - 15:46:11
Django version 1.8, using settings 'main.settings'
Starting development server at http://127.0.0.1:8000/
Quit the server with CONTROL-C.
```

**Comment by nicolasgrasset** [comment]
Good catch @fsto, Django 1.9 changed the syntax and transmission isn't supporting it yet :(

**Comment by nicolasgrasset** [comment]
@fsto the issue is transmissions 0.2.1 actually. It is rolling back a fix of `transmissions==0.1.8.1` which was getting ready for Django 1.9.

I'll merge the 2

**Comment by nicolasgrasset** [comment]
Finally release version 0.2.2 which may require you to be more specific in imports.

It does not support Django 1.10 yet because of https://github.com/5monkeys/django-enumfield/issues/39 so we'll have to see how we fix that soon
